lib/kptyprocess.cpp

```diff
@@ -139,8 +142,6 @@ void KPtyProcess::setupChildProcess()
 
     if (d->ptyChannels & StderrChannel)
         dup2(d->pty->slaveFd(), 2);
-
-    KProcess::setupChildProcess();
 }
```

Comment:
Not sure exactly what you are looking for here.

Reply:
I think this line is removed because `QProcess::setupChildProcess` is removed in Qt 6? It's better to mention the reason for this removal in a commit message.

Reply:
Yes its removed due to Qt 6 removing QProcess::setupChildProcess.  Not sure how I would add that to q commit message at this point.

Reply:
`git rebase` allows rewriting commit messages. It's okay to skip that step, as discussions here can also be a good reference.